### PR TITLE
feat(i18n): Add Spanish translation of comments

### PR DESCRIPTION
**Pull request by itsMeNicoG**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

I've added a spanish folder and the respective comments.js file to the curriculum/dictionaries directory.

### CODE REVIEW

File: curriculum/dictionaries/spanish/comments.js [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+    text: 'Renderiza un input, button, y ul debajo de esta línea'
+  },
+  {
+    id: 'yq81wg',
+    text: 'Renderiza el Provider debajo de esta línea'
+  },
+  {
+    id: 'kxio9j',
+    text:
+      'responseFromServer es establecido a true para representar una respuesta no satisfactoria del servidor'
```
Comment (RandellDawson) [inline curriculum/dictionaries/spanish/comments.js:310]: The English text for this key is:
> responseFromServer is set to false to represent an unsuccessful response from a server

I believe you meant to put `false` in this line:
```suggestion
      'responseFromServer es establecido a false para representar una respuesta no satisfactoria del servidor'
```

File: curriculum/dictionaries/spanish/comments.js [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+    text: 'Esto almacenará el conjunto'
+  },
+  {
+    id: 'x1djjr',
+    text:
+      'Usa console.clear() en la siguiente línea para limpiar la consola del navegador.'
+  },
+  {
+    id: '22ta95',
+    text: 'Usa console.log() para imprimir la variable de salida.'
```
Comment (RandellDawson) [inline curriculum/dictionaries/spanish/comments.js:414]: The English text for this id is:
> Use console.log() to print the output variable.

`output` is a variable name and should not be translated.

File: curriculum/dictionaries/spanish/comments.js [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+    text: 'Obtiene las pestañas antes de la pestaña'
+  },
+  {
+    id: 'bfd23c',
+    text: `Dada una función (representando el tipo de té) y el número de tazas necesarias, la 
+    siguiente función retorna un arreglo de cadenas de texto (cada una representando un tipo específico de té).`
+  },
+  {
+    id: 'ead98i',
+    text: 'Variable de conteo global:'
```
Comment (RandellDawson) [inline curriculum/dictionaries/spanish/comments.js:235]: The English text is:
> Global count variable:

The variable named `count` should not be translated.

File: curriculum/dictionaries/spanish/comments.js [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+    text: 'Define una acción aquí:'
+  },
+  {
+    id: 'tegkqa',
+    text: 'Define el componente Container aquí:'
+  },
+  {
+    id: 'b5oihn',
+    text:
+      'Define el contador reductor que aumentará o disminuirá el estado en función de la acción que reciba'
```
Comment (RandellDawson) [inline curriculum/dictionaries/spanish/comments.js:148]: The English text is:
> Define the counter reducer which will increment or decrement the state based on the action it receives

The word `counter` should not be translated as it represents a code variable.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@itsMeNicoG I noticed a few issues.  I documented a few words that should not have been translated.

@RafaelDavisH @raisedadead If I am not mistaken, the folder name should be `espanol` instead of `spanish`.  If so, can you reply back to this PR to let the contributor know?

Thank you.

**Comment by itsMeNicoG** [comment]
Thanks for the review. I've now corrected the variable names that were translated by mistake. I'm still waiting for confirmation on the folder's name and can update it if necessary.

**Comment by RafaelDavisH** [comment]
@itsMeNicoG thanks for the quick changes. Please do change the folder to "espanol". We are following this convention for consistency. Thanks again. 🙏

**Comment by itsMeNicoG** [comment]
@RafaelDavisH folder has now been renamed as required. Please let me know if there are any other changes to be made.

**Review by RafaelDavisH — COMMENTED** [review]
LGTM 🥳 Thanks for making these translations and changes @itsMeNicoG 🎉

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@raisedadead I am good with these changes now.

**Pull request merged**